### DISCUSSION TITLE
enhancement: retarget pull requests

### DISCUSSION THREAD

**Discussion by boneskull** [community/community#7887] [Pull Requests]
Once a user has created a PR, it’s not possible to change the target (base) of the PR. This can be helpful if:

- A mistake has been made
- The base branch itself has been merged into a third branch. This happens with stacked PRs and it is a big headache to keep in sync (and I believe is part of why graphite.dev exists). Since you can’t retarget, the PR must be merged into the base branch which then itself must be merged (again) into the third branch. This can then only be solved with a manual merge or another PR. (“merge” could be “rebase”)
- Closing a PR causes in lieu of another causes discoverability & paper-trail issues

**Comment by k4b7** [comment]
I hope I'm not misunderstand this feature request, but I think this may already be available: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request.

**Reply by Febbe** [reply]
Yes, it is possible, but the branch must be cherry-picked either way. There would be an improvement, if the commits are automatically cherry-picked.
